pages/community/gsoc.js

```diff
@@ -0,0 +1,100 @@
+import GenericLayout from '../../components/layout/GenericLayout'
+import Heading from '../../components/typography/Heading'
+import Paragraph from '../../components/typography/Paragraph'
+
+export default function GSoCPage() {
+
+  return (
+      <div className="py-12 overflow-hidden">
+        <div className="relative max-w-xl mx-auto px-4 sm:px-6 lg:px-8 lg:max-w-screen-xl">
+          <div className="relative">
+            <Heading level="h1" typeStyle="heading-lg">
+              Google Summer of Code at AsyncAPI 
+            </Heading>
+            <Paragraph>
+            The intention of this page is to list all information that explains how AsyncAPI Initiative participates in Google Summer of Code (GSoC) in 2022.
+            </Paragraph>
+          </div>
+
+          <div className="relative mb-8 lg:mt-8">
+              <Heading level="h2" typeStyle="heading-md-semibold">
+              A bit of history
+              </Heading> 
+              <Paragraph className="mt-4 lg:pr-4">
+              AsyncAPI already participated in GSoC in 2021. We were not accepted as organization, but it was also a time when we were not yet part of the Linux Foundation. Therefore some of AsyncAPI maintainers pushed their ideas through Postman (that was accepted for GSoC) because they were Postman employees working on AsyncAPI. So far so good?
+              <br></br>
+              We mentored 5 mentees. All of them successfully completed GSoC. As a result, all of them became members of our Technical Steering Committee and also presented their solutions at AsyncAPI Conference 2021:
+            
+              <ul>
```

Comment:
This unordered list will not show the bullet points due to the basic CSS implemented in the repo. It automatically adds the `list-style:none` property to every `<ul>` tag. You have to add the classes to this tag in order to show it. They are `className="list-disc pl-8"`